drive_interface:
  ros__parameters:
    front_left: 3
    front_right: 2
    back_left: 1
    back_right: 0

    servo_min_pwm: 1000
    servo_max_pwm: 2000

    topic_sequence: ["back_left","back_right","front_left","front_right"]
    topic_name: "motor_control"